packages/components/TestimonialSection.tsx

```diff
@@ -63,8 +73,8 @@ interface TestimonialCardProps {
 
 function TestimonialCard({ name, testimonial, img }: TestimonialCardProps) {
   return (
-    <Box h="400" rounded="lg" shadow="lg" p={2}>
-      <Flex justifyContent="space-between" mb="1">
+    <Box h="300px" rounded="lg" shadow="lg" p={2}>
+      <Flex justifyContent="space-between" mb="1" w="300px">
```

Comment:
minor: sempre bom evitar medidas fixas como `px, cm, pt, ...`. Sera que conseguimos alterar ou removver as medidas do `h` e `w`?

Reply:
acho q vc me deu uma dica dessas antes mas n to lembrando . consegui tirar o h aq